Some configuration options have (temporarily?) disappeared, so patch in
the right values manually.
--- setup/CONFIGUR.i.orig	2018-07-14 11:35:23.000000000 -0500
+++ setup/CONFIGUR.i	2018-09-03 10:06:49.000000000 -0500
@@ -8,3 +8,9 @@
 	see CNFGDLFT.i and CNFGOPTS.i for things
 		that can go here.
 */
+
+#define UseCmndLine 1
+#define kMaintainerName "MacPorts"
+#define kStrHomePage "https://www.macports.org"
+#define cur_ide gbk_ide_xcd
+#define ide_vers @IDE_VERS@
